charter.html

```diff
@@ -0,0 +1,330 @@
+      users or compromise web services.
+    </p>
+    <p>
+      The group welcomes participation from anti-fraud service providers,
+      common targets of unwanted traffic, browser vendors, privacy advocates,
+      web application developers, web hosting and cloud service providers, and
+      other interested parties.
+    </p>
+    <p>
+      The Community Group will discuss issues that server operators (network
+      and web server admins) and anti-fraud service providers face in this
+      area, as well as ideas for new web features and APIs intended to be
+      implemented in browsers or similar user agents.
+    </p>
+    <h2 id="scope-of-work">
+      Scope of Work
+    </h2>
+    <p>
+      The Community Group will discuss and document various forms of fraudulent
+      activity on the web, to establish a foundation for developing web APIs
+      that assist in our mission. The focus is to enable scaled anti-fraud and
+      solutions that are applicable across the ecosystem. 
+    </p>
+    <h3 id="out-of-scope">
+      Out of Scope
+    </h3>
+    <p>
+      Carve-outs (exceptions to restrictions/limitations) for other web
+      platform work should be brought up in the appropriate group, rather than
+      as part of the Anti-Fraud CG. Features which don’t interact with the
+      browser (or similar user agent) should be proposed in other standards
+      bodies (IETF, etc).
+    </p>
+    <h2 id="meetings">
+      Meetings
+    </h2>
+    <p>
+      Meetings will be held biweekly on {TODO: INSERT DATE}, lasting for one
+      hour. Meetings will be run by one of the Chairs and will request scribes
+      to take meeting notes, which will be posted in the
+      <a href="https://github.com/antifraudcg/meetings">Meetings</a> repository
+       after the meeting. Any action items from the meeting will also be posted
+       to the Community Group mailing list to bring them to the attention of
+       people who did not attend.
+    </p>
+    <p>
+      Infrequently, face-to-face meetings will be held for the Anti-Fraud
+      Community Group, allowing for more collaboration and iteration between
+      Community Group members.
+    </p>
+    <h2 id="deliverables">
+      Deliverables
+    </h2>
+    <h3 id="work-items">
+      Current Work Items
+    </h3>
+    <p>
+      There are currently no work items that have been adopted by the Community
+      Group. See <a href="#proposals">Proposals</a> for the process of adopting
+      a proposal into the Community Group.
+    </p>
+    <p>
+      Once a work item is ready for standardization and there is
+      <a href="#decision">consensus</a> that the work item is ready to
+      be migrated to a standards body, the Chairs will collaborate with the
+      Community Group to prepare a Community Group Report on the item, and with
+      the Editors of the item to forward it and the Report to the appropriate
```

Comment:
It is not clear to me how "it" (the work item) is distinguished from the CG Report. Would it suffice to say that the group prepares a Report on the topic and then disseminates it to the relevant parties?